homework4/task3.js

```diff
@@ -0,0 +1,8 @@
+function arrayDivisionFactory(num) {
```

Comment:
INCORRECT

all code should be inside `arrayDivisionFactory` and without changing `divide5` and `divide3`